src/providers/caching-gas-provider.ts

```diff
@@ -12,7 +12,8 @@ import { GasPrice, IGasPriceProvider } from './gas-price-provider';
  * @class CachingV3SubgraphProvider
```

Comment:
All those gas-providers related source files should have no changes in this PR. It feels like the PR isn't fully merged from main.

Reply:
Seems like prettier / eslint changes